doc/getting_started/up2.rst

```diff
@@ -29,10 +30,10 @@ be plugged into another PC for debugging. Tool like minicom or screen
 can help to connect.
 
 Software setup
-==============
+**************
 
 Setting up the operating system
--------------------------------
+===============================
 
 Download the compressed Clear installer image from
 `*here* <https://download.clearlinux.org/releases/21260/clear/clear-21260-installer.img.xz>`__
```

Comment:
We avoid using link text such as "here".  Instead write this as:

    Download the compressed Clear installer image from the `Clear Linux release downloads site
    <https://download.clearlinux.org/releases/21260/clear/clear-21260-installer.img.xz>`__

Won't this specific release link be a problem?  Should this link be to the "releases" folder and we tell them to download the latest?